security/uacme/Portfile

```diff
@@ -0,0 +1,54 @@
+PortSystem          1.0
```

Comment:
The Portfile is missing the modeline - that should be the first line in every Portfile:

```
# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4

PortSystem          1.0
```

That's used by editors to set the right formatting and spacing for Portfiles.